controllers/mover/restic/builder.go

```diff
@@ -119,6 +119,8 @@ func (rb *Builder) FromSource(client client.Client, logger logr.Logger,
 	saHandler := utils.NewSAHandler(client, source, isSource, privileged,
 		source.Spec.Restic.MoverServiceAccount)
 
+	isImmediate := source.Spec.Immediate == nil
```

Comment:
```suggestion
	isImmediate := source.Spec.Immediate != nil
```

?

Reply:
Isn't this variable a boolean, why is it being compared to nil?

Reply:
The instantiation of `Immediate` is a pointer, so here I am making the default `true`. I couldn't see another way to do this so I am all ears if there is a better way.

Reply:
I see, but the actual value is never consumed anywhere. It should be something like:

```suggestion
	isImmediate := true
	if source.Spec.Immediate != nil {

		isImmediate = * source.Spec.Immediate

	
}
```